package Message;

message ChatRequest{
	optional int64		id 			= 1;
	optional string		message 	= 2;
	optional int64		target		= 3;
	optional string		targetName	= 4;
};

message ChatResponse {
	optional int32		result 		= 1;
};

message ChatNotify{
	optional int64		id 			= 1;
	optional string		message 	= 2;
	optional string		name 		= 3;
	optional int32		vip_level	= 4;
	optional string		camp		= 5;
	optional string		knights		= 6;
	optional int64		targetID	= 7;
	optional string		targetName	= 8;
};

